exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,12 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var porcentageStudents =
+  (numberOfStudents * 100) / (numberOfStudents + numberOfMentors);
+var porcentageMentors =
+  (numberOfMentors * 100) / (numberOfStudents + numberOfMentors);
+
```

Comment:
since you use the **numberOfStudents + numberOfMentors** twice, it can be assigned to a variable like
var total = numberOfStudents + numberOfMentors